.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,21 @@
+name: golangci-lint
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
+          version: latest
```

Comment:
Please check your editor, EOL symbol is required.

Reply:
OK.
It seemed that `golangci/golangci-lint-action@v2` is not allowed to run in this repository,any suggestion?

Error info:
```
golangci/golangci-lint-action@v2 is not allowed to be used in apache/apisix-ingress-controller. Actions in this workflow must be: within a repository owned by apache, created by GitHub, verified in the GitHub Marketplace or match the following: */*@[a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9][a-f0-9]+, AdoptOpenJDK/install-jdk@*, al-cheb/configure-pagefile-action@*, apache/*, burrunan/gradle-cache-action@*, bytedeco/javacpp-presets/.github/actions/*, chromaui/action@*, codecov/codecov-action@*, conda-incubator/setup-miniconda@*, container-tools/kind-action@*, dawidd6/action-download-artifact@*, gr2m/twitter-together@*, gradle/wrapper-validation-action@*, julia-actions/julia-runtest@*, julia-actions/setup-julia@*, msys2/setup-msys2@*, peaceiris/actions-gh-pages@*, peaceiris/actions-hugo@*, peter-evans/create-pull-request@*, potiuk/cancel-workflow-runs@*, r-lib/actions/*, ruby/setup-ruby@*, scacap/action-surefire-report@*, shivammathur/setup-php@*, shogo82148/actions-setup-perl@*,...
```

Reply:
You may need to download golangci-lint and run it manually, please refer to apisix-dashboard and see how it works.